### DISCUSSION TITLE
Edit IP address and assign to specific device-interface - cannot filter by device

### DISCUSSION THREAD

**Discussion by sliddjur** [netbox-community/netbox#14561] [General]
Hello, when I go to edit an IP address, which is currently unassigned. I want to assign it to a specific device interface - but IMO it is  not so easy. Because when searching for the interface name, I cannot filter for the specific device I am looking for.

Can a device filter be added here when editing IP-addresses?

![image](https://github.com/netbox-community/netbox/assets/10002057/3a0d52e5-959b-4b88-9611-e9cbd0fa87af)

**Comment by candlerb** [comment]
Did you try clicking the filter icon to the right?

![image](https://github.com/netbox-community/netbox/assets/44789/7e03bf77-00e1-4906-b3db-e6723cf648b6)

It's clunky, certainly compared to how it used to work in older versions of Netbox, but you should be able to enable filtering on Device, and then select the device of interest, and then see only interfaces from that device.

![image](https://github.com/netbox-community/netbox/assets/44789/3f694bd5-6d3e-40d5-8c48-b424487efc94)
